src/test/java/edu/oakland/test/Display01/DisplayGpsReceiverImplementation/DisplayGpsReceiverImplementationTests.java

```diff
@@ -0,0 +1,21 @@
+package edu.oakland.test.Display01.DisplayGpsReceiverImplementation;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.oakland.production.display01.DisplayGpsReceiverImplementation;
+import java.lang.IllegalArgumentException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+
+
+@DisplayName ("Display GPS Receiver Implementation")
```

Comment:
For sake of standardization to make it easier to read and search the logs, can we change this to `DisplayGpsReceiverImplementation Unit Tests`? Thank you!
```suggestion
@DisplayName ("DisplayGpsReceiverImplementation Unit Tests")
```